go/memcached-operator/config/manifests/bases/memcached-operator.clusterserviceversion.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
-    operators.operatorframework.io/builder: operator-sdk-v1.1.0
+    operators.operatorframework.io/builder: operator-sdk-v1.0.0+git
```

Comment:
Hi @jmrodri @theishshah, 

See that by mistake. May it was missing a rebase with the master, the script was merged with changes in the samples when should not. I will do a PR to fix the samples.

Reply:
I saw the samples. Thought it contained the 1.1 changes